components/docs-chef-io/content/automate/ha_backup_restore_aws_efs.md

```diff
@@ -121,6 +121,16 @@ Configure the OpenSearch `path.repo` attribute.
     chef-automate backup create
     ```
 
+### Pre-Restore Validation
+
+Before executing the actual restore command, it is recommended to use the --verify-restore-config flag. This flag performs a pre-check to ensure that all backup and restore-related configurations are correctly set.
```

Comment:
Replace it with: Run the restore command with the --verify-restore-config flag to perform a pre-check. This ensures all backup and restore-related configurations are set correctly before execution.